### PR TITLE
adding more installation details to the readme ⚡ 

### PR DESCRIPTION
**Pull request by manuelduarte077**
a little help with the readme, add some installation steps I hope it can be useful 💯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-# flutter-clean-architecture
+## Introduction
+
+This [VS Code](https://code.visualstudio.com/) extensions will make following clean architecture guidelines easier when using Flutter for development.
```
Comment (marcossevilla) [inline README.md]: Change `extensions` to `extension`.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
-# flutter-clean-architecture
+## Introduction
```
Comment (marcossevilla) [inline README.md:3]: Include `h1` "**Flutter Clean Scope**" before "**Introduction**".

File: README.md (hunk 1 above)
Comment (marcossevilla) [reply README.md]: Change Clean Architecture to uppercase in first character.

**Review by marcossevilla — CHANGES_REQUESTED** [review]
Nice PR, just some writing reviews.

File: README.md (hunk 1 above)
Comment (manuelduarte077) [reply README.md]: Done, suggested changes resolved

File: README.md (hunk 2 above)
Comment (manuelduarte077) [reply README.md:3]: Resolved

**Comment by manuelduarte077** [comment]
I think the suggested changes have already been resolved, thank you very much for the reviews.

File: README.md (hunk 2 above)
Comment (marcossevilla) [reply README.md:3]: `h1` in Markdown is used with `#`, it should be `# Flutter Clean Scope`.

**Review by marcossevilla — CHANGES_REQUESTED** [review]
Small syntax change in extension title (**Flutter Clean Scope**).

File: README.md (hunk 2 above)
Comment (manuelduarte077) [reply README.md:3]: Resolved `# Flutter Clean Scope`

**Review by manuelduarte077 — COMMENTED** [review]
Resolved

**Review by marcossevilla — APPROVED** [review]
Thanks!

**Pull request merged**